examples/react-template/screens/Column.tsx

```
@@ -19,61 +19,121 @@ import {
-          <ColumnsItem><Text>Test</Text></ColumnsItem>
-          <ColumnsItem><Text>Test</Text></ColumnsItem>
-        </Columns>
-      </Box>
+      <Divider/>
+      <Title level={TitleLevels.THREE}> Scollable and Fullblead</Title>
+      <Text> Columns is scrollable (Add mobile to force columns to stay in one row) </Text>
+      <Columns scrollable mobile>
+        {
+          Object.values([1,2,3,4,5,6,7,8,9,10,11,12]).map( (colSize, index) => {
+            return (
+              <ColumnsItem size={colSize}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)